src/main/java/com/afwsamples/testdpc/common/Util.java

```diff
@@ -354,4 +355,125 @@ public static String constantToString(Class<?> clazz, String prefix, int value)
+        break;
+      case SecurityLog.TAG_KEYGUARD_SECURED:
+        eventTag = "KEYGUARD_SECURED";
+        break;
+      case SecurityLog.TAG_OS_STARTUP:
+        eventTag = "OS_STARTUP";
+        break;
+      case SecurityLog.TAG_OS_SHUTDOWN:
+        eventTag = "OS_SHUTDOWN";
+        break;
+      case SecurityLog.TAG_LOGGING_STARTED:
+        eventTag = "LOGGING_STARTED";
+        break;
+      case SecurityLog.TAG_LOGGING_STOPPED:
+        eventTag = "LOGGING_STOPPED";
+        break;
+      case SecurityLog.TAG_MEDIA_MOUNT:
+        eventTag = "MEDIA_MOUNT";
+        break;
+      case SecurityLog.TAG_MEDIA_UNMOUNT:
+        eventTag = "MEDIA_UNMOUNT";
+        break;
+      case SecurityLog.TAG_LOG_BUFFER_SIZE_CRITICAL:
+        eventTag = "LOG_BUFFER_SIZE_CRITICAL";
+        break;
+      case SecurityLog.TAG_PASSWORD_EXPIRATION_SET:
+        eventTag = "PASSWORD_EXPIRATION_SET";
+        break;
+      case SecurityLog.TAG_PASSWORD_COMPLEXITY_SET:
+        eventTag = "PASSWORD_COMPLEXITY_SET";
+        break;
+      case SecurityLog.TAG_PASSWORD_HISTORY_LENGTH_SET:
+        eventTag = "PASSWORD_HISTORY_LENGTH_SET";
+        break;
+      case SecurityLog.TAG_MAX_SCREEN_LOCK_TIMEOUT_SET:
+        eventTag = "MAX_SCREEN_LOCK_TIMEOUT_SET";
+        break;
+      case SecurityLog.TAG_MAX_PASSWORD_ATTEMPTS_SET:
+        eventTag = "MAX_PASSWORD_ATTEMPTS_SET";
+        break;
+      case SecurityLog.TAG_KEYGUARD_DISABLED_FEATURES_SET:
+        eventTag = "KEYGUARD_DISABLED_FEATURES_SET";
+        break;
+      case SecurityLog.TAG_REMOTE_LOCK:
+        eventTag = "REMOTE_LOCK";
+        break;
+      case SecurityLog.TAG_WIPE_FAILURE:
+        eventTag = "WIPE_FAILURE";
+        break;
+      case SecurityLog.TAG_KEY_GENERATED:
+        eventTag = "KEY_GENERATED";
+        break;
+      case SecurityLog.TAG_KEY_IMPORT:
+        eventTag = "KEY_IMPORT";
+        break;
+      case SecurityLog.TAG_KEY_DESTRUCTION:
+        eventTag = "KEY_DESTRUCTION";
+        break;
+      case SecurityLog.TAG_USER_RESTRICTION_ADDED:
+        eventTag = "USER_RESTRICTION_ADDED";
+        break;
+      case SecurityLog.TAG_USER_RESTRICTION_REMOVED:
+        eventTag = "USER_RESTRICTION_REMOVED";
+        break;
+      case SecurityLog.TAG_CERT_AUTHORITY_INSTALLED:
+        eventTag = "CERT_AUTHORITY_INSTALLED";
+        break;
+      case SecurityLog.TAG_CERT_AUTHORITY_REMOVED:
+        eventTag = "CERT_AUTHORITY_REMOVED";
+        break;
+      case SecurityLog.TAG_CRYPTO_SELF_TEST_COMPLETED:
+        eventTag = "CRYPTO_SELF_TEST_COMPLETED";
+        break;
+      case SecurityLog.TAG_KEY_INTEGRITY_VIOLATION:
+        eventTag = "KEY_INTEGRITY_VIOLATION";
+        break;
+      case SecurityLog.TAG_CERT_VALIDATION_FAILURE:
+        eventTag = "CERT_VALIDATION_FAILURE";
+        break;
+      default:
+        eventTag = "UNKNOWN(" + eventId + ")";
+    }
+    return eventTag;
+  }
+
+  public static void printData(StringBuilder sb, Object data) {
```

Comment:
Could we leave this in SecurityLogsFragment and call it statically from there? then it's more obvious that this method is only related to security logs?

Reply:
Sounds good.
Done